### PR TITLE
Price displays to 4 decimal places

### PR DESCRIPTION
**Pull request by NickNewlands**
story: [AB#11143](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11143)
task: [AB#12043](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12043)

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -35,7 +35,9 @@ rules:
   no-empty-rulesets: 2
   no-extends: 0
   no-ids: 2
-  no-important: 2
+  # https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/core/utilities/_typography.scss
+  # "Utility classes are allowed to use !important;"
+  no-important: 0
   no-invalid-hex: 2
```
Comment (NickNewlands) [inline .sass-lint.yml:41]: There was some code already in the codebase that was stopping me from committing without this. It is now the same as the dev branch.

File: app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json [hunk 2]
```diff
@@ -48,6 +48,12 @@
     }
   },
   "errorMessages": {
+    "PriceRequired": "Enter a price",
+    "PriceMustBeANumber": "Price must be a number",
+    "PriceMoreThan4dp": "Price must be to a maximum of 4 decimal places",
+    "PriceGreaterThanOrEqualToZero": "Price cannot be negative",
+    "PriceLessThanMax": "Maximum price exceeded, enter a lower price",
+    "PriceGreaterThanListPrice": "You’ve entered a price that is more than the list price, enter a lower price",
```
Comment (bjss-chris) [inline app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: It seems odd that these messages don't exist in this version. Are we not doing in validation on the price or are these messages define somewhere else?

File: app/pages/sections/order-items/catalogue-solutions/order-item/controller.test.js [hunk 3]
```diff
@@ -131,7 +131,6 @@ describe('catalogue-solutions order-item controller', () => {
         formData: { quantity: '100' },
         validationErrors: {},
       });
-
```
Comment (bjss-chris) [inline app/pages/sections/order-items/catalogue-solutions/order-item/controller.test.js:134]: I guess we can omit this change.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Just a couple of small requests and one question.

File: app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json (hunk 2 above)
Comment (NickNewlands) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: Ahh just a bad merge, I think my compare used the wrong lines.

Comment (bjss-chris) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: I think the same thing might have happened again – it looks we've still got the price error messages.

Comment (NickNewlands) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: I removed these properties from lower down in the object. I think it should be fine.

Comment (NickNewlands) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: And it should make further merges more seamless.

Comment (bjss-chris) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: If you have a look at the whole file you should be able to see that they're still there, above the quantity error messages.

Comment (bjss-chris) [reply app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json:56]: Oh, hang on a sec. Have they just been reordered for some reason? Let me check.

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**